composer.json

```diff
@@ -139,7 +139,7 @@
         {
             "_comment": "The PhingDrushTask module does not currently have a D8 release and is not available on https://packages.drupal.org/8. See https://www.drupal.org/node/2817131.",
             "type": "vcs",
-            "url": "git@github.com:pfrenssen/phingdrushtask.git"
+            "url": "https://git.drupal.org/project/phingdrushtask.git"
```

Comment:
Since it's now in the official repo, shouldn't we be able to directly remove it in all? I mean the repo definition! It should be automatically picked.